### PR TITLE
ENG-7552: Rip out spUniqueId (rebased)

### PR DESCRIPTION
**Pull request by afrogers**

### CODE REVIEW

File: src/frontend/org/voltdb/iv2/MpScheduler.java [hunk 1]
```diff
@@ -542,9 +543,4 @@ private static MpProcedureTask instantiateNpProcedureTask(Object...params)
         }
         return null;
     }
-
-    @Override
-    public void setMaxSeenUniqueId(final long maxSeenUniqueId) {
```
Comment (nshi) [inline src/frontend/org/voltdb/iv2/MpScheduler.java]: Why is this removed?

**Comment by nshi** [comment]
Review completed.

File: src/frontend/org/voltdb/iv2/MpScheduler.java (hunk 1 above)
Comment (afrogers) [inline src/frontend/org/voltdb/iv2/MpScheduler.java]: So this was added as part of Ariel's introduction of spUniqueIds. It looks
like it has some potentially desirable consequences for making uniqueId
more solid on repair, except in a way that really doesn't appear strictly
necessary. Last generated uniqueId gets updated on repair anyway if there
are still transaction messages in the log, and we're no longer relying on
uniqueId for anything but stats and making sure transactions are who they
say they are.

On Thu, Feb 26, 2015 at 9:49 AM, Ning Shi <notifications@github.com> wrote:

> In src/frontend/org/voltdb/iv2/MpScheduler.java
> <https://github.com/VoltDB/voltdb/pull/2070#discussion_r25431509>:
>
> > @@ -542,9 +543,4 @@ private static MpProcedureTask instantiateNpProcedureTask(Object...params)
> >          }
> >          return null;
> >      }
> > -
> > -    @Override
> > -    public void setMaxSeenUniqueId(final long maxSeenUniqueId) {
>
> Why is this removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/VoltDB/voltdb/pull/2070/files#r25431509>.
>



-- 

*[image: VoltDBLogoEmail]* <http://www.voltdb.com/>
Alex Rogers, Software Engineer
*VoltDB, Inc.*
*E: *arogers@voltdb.com

*voltdb.com* <http://www.voltdb.com/>
Follow VoltDB [image: twitter] <https://twitter.com/VoltDB> [image:
facebook] <https://www.facebook.com/VoltDB> [image: linkedin]
<https://www.linkedin.com/company/voltdb>

<http://www.linkedin.com/company/voltdb>
